cloudigrade/util/models.py

```diff
@@ -12,3 +12,10 @@ class BaseModel(PolymorphicModel):
     class Meta:
         abstract = True
         ordering = ('created_at',)
+
+
+class BaseModel(models.Model):
+    """Abstract model to add automatic created_at and updated_at fields."""
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
I think `BaseModel` also needs to include the inner `class Meta` copied from `BasePolymorphicModel`.